__tests__/app.test.js

```diff
@@ -316,7 +316,151 @@ describe("app", () => {
+    const requestBody2 = { inc_votes: 15 };
+    const expectedVotes = 3;
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(200)
+    .then((response) => {
+      return request(app)
+      .patch(`/api/articles/${testArticleId}`)
+      .send(requestBody2)
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toHaveProperty("votes", expectedVotes);
+      });
+    });
+    
+  });
+  test("404: responds with a status of 404 when the article is not found", () => {
+    const testArticleId = 40000;
+    const requestBody = { inc_votes: 5 };
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe("No article found for article_id: 40000")
+    });
+  });
+  test("400: responds with a status of 400 when the article id is an invalid number", () => {
+    const testArticleId = "Hello";
+    const requestBody = { inc_votes: -1 };
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe("Invalid input");
+    });
+  });
+  test("400: responds with a status of 400 when the inc_votes property is missing", () => {
+    const testArticleId = 4;
+    const requestBody = { };
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe("An inc_votes property is required");
+    });
+  });
+  test("400: responds with a status of 400 when the inc_votes property is an invalid number", () => {
+    const testArticleId = 4;
+    const requestBody = { inc_votes: "cats" };
+    return request(app)
+    .patch(`/api/articles/${testArticleId}`)
+    .send(requestBody)
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe("Invalid input");
+    });
+  });
+ });
+ describe("DELETE /api/comments/:comment_id", () => {
+  test("404: responds with a status of 404 when the comment is not found", () => {
+    const testCommentId = 40000;
+    return request(app)
+    .delete(`/api/comments/${testCommentId}`)
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe("No comment found with comment_id: 40000")
+    });
+  });
+  test("400: responds with a status of 400 when the comment id is an invalid number", () => {
+    const testCommentId = "hello";
+    return request(app)
+    .delete(`/api/comments/${testCommentId}`)
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe("Invalid input");
+    });
+  });
+  test("204: deleting an existing comment responds with 204 and no content", () => {
+    const testCommentId = 1;
+    return request(app)
+    .delete(`/api/comments/${testCommentId}`)
+    .expect(204)
+    .then((response) => {
+      expect(response.body).toEqual({});
```

Comment:
When we are deleting a comment it's ok to just send a 204 status code, we don't need to send any sort of object back.